exporters/zipkin/build.gradle

```diff
@@ -0,0 +1,56 @@
+plugins {
+    id "java"
+    id "maven-publish"
+
+    id "ru.vyarus.animalsniffer"
+}
+
+description = 'OpenTelemetry - Zipkin Exporter'
+ext.moduleName = "io.opentelemetry.exporters.zipkin"
+
+dependencies {
+    compileOnly libraries.auto_value
+
+    api project(':opentelemetry-sdk')
+
+    annotationProcessor libraries.auto_value
+
+    implementation libraries.guava
```

Comment:
I'd take out guava. unless it is used somehow underneath, the types used in the code below aren't pulling weight to pin this. I know you're surprised that I, of all people, am making this comment

Reply:
I'll move it to a `testImplementation` and clean up the very few usages.